### ISSUE TITLE
OpenSUSE color suggestion

### ISSUE THREAD

**Issue by ElectricGestalt** [godlyranchdressing/United-GNOME#76]
The two tone Orange/Purple and Blue/Gold look really great for the Ubuntu and Fedora themes. I notice you opted for Neon Green in the OpenSUSE theme. Have you considered Red? This picture is the inspiration for my suggestion.

![Chameleon Red](https://i.imgur.com/ffQ15Pj.png)

**Comment by godlyranchdressing** [comment]
The red does look pretty good!
![screenshot from 2017-05-22 22-18-30](https://cloud.githubusercontent.com/assets/27529229/26335837/1a471f50-3f1b-11e7-902c-95e60297f2c3.png)
But Ubuntu, Fedora and openSUSE's palettes are pretty much set in stone since they're all taken from official brand guidelines documents. [I found the sea green color here](https://opensuse.github.io/branding-guidelines/). The others I'm more open to changing.

That is some indeed some **_neon green_** now that you mention it though. I decreased the amount that openSUSE's accent color is lightened by for the slider and progress bars because I just checked them and their almost blinding.
